### ISSUE TITLE
How do I expose implementation of Dummy when using cfg(test)?

### ISSUE THREAD

**Issue by SpyMachine** [cksac/fake-rs#53]
I am using Dummy to create an object for testing so I've used `[cfg_attr(test, derive(Dummy)]` for the struct I want to fake. I have a "domain" crate where I have my models and then an "http_api" crate which depends on "domain". When I try to get a fake instance of my model struct, I get an error saying that Dummy is not implemented. Is there a way to "expose" that I've implemented the trait here that I'm missing or is this not possible when using cfg(test)? It works perfectly fine if I remove the #[cfg_attr(test, derive(Dummy)] from the struct and just derive it normally.

```
error[E0277]: the trait bound `domain::models::api_requests::PutRequest: Dummy<Faker>` is not satisfied
   --> http_api/src/local/filters.rs:154:48
    |
154 |         let request: PutRequest = Faker.fake();
    |                                                ^^^^ the trait `Dummy<Faker>` is not implemented for `domain::models::api_requests::PutRequest`
```

**Comment by cksac** [comment]
Tested ok, have your test inside `#[cfg(test)]` module?
```rust
use fake::{Dummy, Fake, Faker};
#[derive(Debug)]
#[cfg_attr(test, derive(Dummy))]
pub struct Foo {
    bar: u32,
}

#[cfg(test)]
mod tests {
    use super::Foo;
    use fake::{Fake, Faker};

    #[test]
    fn it_works() {
        println!("{:?}", Faker.fake::<Foo>())
    }
}
```

**Comment by SpyMachine** [comment]
Right, so this woks for me too. The problem arises when I try and "Fake" a struct in one crate from another crate. It seems like the implementation of Dummy is not publicly exposed.

**Comment by SpyMachine** [comment]
Sorry, this was due to my misunderstanding of what happens when you run `cargo test`. I was under the impression that when working with a workspace, all the crates will be built with the test configuration for all tests but it appears. when running unit tests for one crate, it does not use a build that has test objects from a dependency crate in the workspace.

For anyone who comes by this in the future, I was able to workaround this by creating a "test" feature and doing something like `#[cfg_attr(any(test, feature="test"), derive(Dummy))]`. Then in the crate that requires the fake object for testing, just reinclude the crate in dev-dependencies with the 'test' feature enabled.
